test/stepDefinitions/baseSteps.js

```diff
@@ -12,6 +12,13 @@ module.exports = {
     await createCaseStartPage.clickStartButton();
   },
 
+  navigateToCreateCaseMultiPage: async function(){
```

Comment:
lets remove this as it has a hardcoded case type and is not very reusable. instead use navigateToCreateCasePage and pre load the data through a (new?) previous step implemented in definitionSteps.js 
either use existing step `Given a case type with multiple pages exist`
or new one with same implementation `Given a case type with multiple pages containing a dynamic fixed list exists`